libraries/stdlib/src/generated/_Filtering.kt

```diff
@@ -1191,14 +1191,14 @@ public inline fun <T> Iterable<T>.takeWhile(predicate: (T) -> Boolean): List<T>
 }
 
 /**
- * Returns a stream containing first elements satisfying the given *predicate*
+ * Returns a stream containing first elements satisfying the given [predicate]
  */
 public fun <T> Stream<T>.takeWhile(predicate: (T) -> Boolean): Stream<T> {
     return TakeWhileStream(this, predicate)
 }
 
 /**
- * Returns a list containing first elements satisfying the given *predicate*
+ * Returns a string containing the first characters that satisfy the given [predicate]
```

Comment:
We do. "which first characters does it return? the first characters that satisfy"